File: python/RUES-Search.py
#rangos de facturacion de una cuenta

import api

# Datos basicos
USR = "elusuario"
PWD = "lacontrasena"
URL = "https://playground.lafactura.co/api2.0"

# Solicitud
solicitud= {
    'RUES':{
        'search':{
            'name':'ejemplo'
         }
    }

# Envio
response  = api.getRUESSearch(URL,USR,PWD,solicitud)
# Resultado
print(response.text)

File: php/RUES-Search.php
<?php
  //Este ejemplo permite buscar una empresa por nombre en el RUES

  include_once("API/api.php");

  $usuario="elusuario"; //coloca en este punto tu usuario
  $contrasena="secretas"; //coloca tu contraseña
  $url="https://playground.lafactura.co/api2.0"; //estea esl la URL de acceso a la api

  //solicita Registro Mercantil del RUES
  $solicitud=[
    "RUES"=>[
        "search"=>[
            "name"=>"ejemplo" //texto a buscar
        ]
    ]
  ];
  //convierto a JSON la solicitud
  $solicitud=json_encode($solicitud,JSON_UNESCAPED_UNICODE | JSON_HEX_QUOT);

  $api=new LF_API_v200($url,$usuario,$contrasena);
  $api->send(LF_SERVICE_GENERAL,$solicitud,LF_METHOD_GET); //metodo GET
  echo $api->response; //resulado de la peticion en JSON

?>
